lua/telescope/actions.lua

```diff
@@ -185,4 +196,14 @@ actions.insert_value = function(prompt_bufnr)
   return entry.value
 end
 
+for k, v in pairs(actions) do
+  actions[k] = setmetatable({}, vim.tbl_extend("force", {
+    __call = function(_, ...)
+      return v(...)
+    end }, action_mt)
+  )
+end
```

Comment:
I had to change your suggestion a little bit. You had: 
```lua
actions[k] = setmetatable({__call = function(_, ...) return k(...) }, action_mt)
```

If I understand metatable correctly these operator overloads must be in the second parameter. Or am I wrong?
Also do i really need a `vim.tbl_extend` here? It did not work without one.

Reply:
Thanks for the review btw. Helped me a lot :smile: